opacclient/opacapp/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,358 @@
+    <string name="web_opac">Web Opac</string>
+    <string name="branch">Vestiging</string>
+    <string name="barcodes">Barcodes</string>
+    <string name="go_search">Zoeken starten</string>
+    <string name="prefs">Instellingen</string>
+    <string name="account">Account</string>
+    <string name="about">Over</string>
+    <string name="about_summary">Over deze app</string>
+    <string name="changelog">Changelog</string>
+    <string name="close">Sluiten</string>
+    <string name="osl">Licentiekennisgevingen</string>
+    <string name="thanks">Medeauteurs en dankzeggingen</string>
+    <string name="developer_settings">Ontwikkelaarsinstellingen</string>
+    <string name="details_head">Details</string>
+    <string name="copies_head">Exemplaren</string>
+    <string name="ret">Inleverdatum</string>
+    <string name="res">Gereserveerd</string>
+    <string name="tocollection">Naar collectie</string>
+    <string name="reserve">Reserveer</string>
+    <string name="lendebook">Leen e-book</string>
+    <string name="doing_res">Reservering uitvoeren…</string>
+    <string name="opac_error_email">S.V.P. voeg e-mail adres toe aan de instellingen.</string>
+    <string name="doing_booking">Lenen uitvoeren…</string>
+    <string name="status_nouser">Je hebt geen logingegevens gespecifieerd voor dit account in de instellingen.</string>
+    <string name="pending_fees">Uitstaande boetes:</string>
+    <string name="valid_until">Account geldig tot:</string>
+    <string name="no_warranty">Alle informatie wordt verstrekt zonder aansprakelijkheid.</string>
+    <string name="lent_head">Geleend</string>
+    <string name="reservations_head">Reserveringen</string>
+    <string name="lent_none">Niets geleend</string>
+    <string name="reservations_none">Niets gereserveerd</string>
+    <string name="cancel_confirm">Reservering echt annuleren?</string>
+    <string name="prolong_confirm">Item echt verlengen?</string>
+    <string name="yes">Ja</string>
+    <string name="no">Nee</string>
+    <string name="cancel">Annuleer</string>
+    <string name="delete">Verwijder</string>
+    <string name="accept">OK</string>
+    <string name="doing_cancel">Reservering wordt geannuleerd…</string>
+    <string name="doing_prolong">Verlenging wordt doorgevoerd…</string>
+    <string name="doing_download">Download information is being retrieved…</string>
+    <string name="reader_needed">To read the e-book, you need a separate app, for example \'Aldiko\'. Click \'Ignore\' if you already have such an app, this message won\'t show up again in this case. Click \'Download\' to open up the Play Store to download Aldiko.</string>
+    <string name="reader_needed_overdrive">To open this medium, you need a separate app, for example OverDrive\'s own app. Click \'Ignore\' if you already have such an app, this message won\'t show up again in this case. Click \'Download\' to open up the Play Store to download the OverDrive app.</string>
```

Comment:
If you do not want to translate something, just omit it from the file. Including English text in Dutch files makes it very hard to find what translations are missing.